_blog/2022-01-04-Website translations.md

```diff
@@ -0,0 +1,48 @@
+
+# Step 3: prepare for automation & integration
+With a translation-ready website and a translation system of choice, the two had to be hooked up. Weblate can't directly read and handle Markdown (MD) files (in which we write most website texts). The tool [mdpo](https://github.com/mondeja/mdpo) came to the rescue, forming a two-directional bridge between our website's MD files and the PO (Portable Object) files that Weblate can handle.
+
+In order to make our lives easier and keep the code history clean, we wanted to strike a balance between automation and manual action. The system now automatically informs and prepares Weblate if our English website texts change. And the website is automatically recreated with new translations, but only when we push new translations from Weblate. [GitHub Actions](https://github.com/features/actions) is the tool allowing us to do this -- it runs mdpo and other tools on a virtual computer, if certain conditions are met).
+
+When exploring potential translation tools and playing with mdpo, we also experimented with the Weblate set-up. So the final stage of setting up Weblate was the easy part.
+
+# Over to you: contributing translations
+We've prepared the grounds. With your help, we hope we can properly (and quickly) localise our website. Contributing as translator is easy: Simply find [the AntennaPod project on Weblate.org](https://hosted.weblate.org/engage/antennapod/), log in (or create an account) and start translating!
+
+If your language is not available yet, please chip in on our forum and let us know which language we should add. Feel free to tag me (@keunes) for a faster follow-up. If you have any questions about a text (e.g. if it is a verb or noun), you can add a comment:
+![Screenshot of the 'Comments' tab on Weblate](assets/images/blog/2022/website-translations_weblate-comment.jpg)
+
+We hope our existing translators will join us in this effort: they know the app, which helps to ensure consistency. But whether you're new to Weblate or haven't used AntennaPod before: we welcome you with open arms! Ask on forum to add language. And don't hesitate to tell us what you think of Weblate.
+
+# An educational experience
+On a personal level, this whole process has been a very educational experience. I was only vaguely familiar with git, Jekyll, Liquid, GitHub actions, Weblate project management and mdpo before starting this endeavour. Many thanks to all those that kindly answered my questions in the different repositories and fora.
```

Comment:
This is the first time you use `I` instead of `we`, so I would mention your name again here

```suggestion
On a personal level, this whole process has been a very educational experience. I (@keunes) was only vaguely familiar with git, Jekyll, Liquid, GitHub actions, Weblate project management and mdpo before starting this endeavour. Many thanks to all those that kindly answered my questions in the different repositories and fora.
```